src/createClient.js

```diff
@@ -15,7 +15,7 @@ function createClient (options) {
     client.ping().then(data => {
       const advert = advertisement.fromServerName(data)
       console.log(`Connecting to server ${advert.motd} (${advert.name}), version ${advert.version}`)
-      // TODO: update connect version based on ping response
+      client.version = data.ping
```

Comment:
I believe this should be advert.version. Also I think it would be a good idea to only set the version if it wasn't already explicitly provided in the options

Reply:
Okay sure

Reply:
done in 5568876